src/html/network-banner.html

```diff
@@ -1,21 +1,35 @@
-<aside class="banner yellow" data-behavior="ropstenBanner" style="display:none">
-  Operating on Ropsten ↔︎ NEAR Testnet
+<aside class="banner yellow" data-behavior="testnetBanner" style="display:none">
+  <span data-behavior="testnetWarning"></span>
 </aside>
-<aside class="banner red" data-behavior="unsupportedNetwork" style="display:none">
-  Unsupported Ethereum Network! Please use Ropsten.
+<aside class="banner red" data-behavior="unsupportedNetworkBanner" style="display:none">
+  <span data-behavior="unsupportedNetwork"></span>
 </aside>
+<div class="fullScreen" data-behavior="unsupportedNetworkBanner" style="display: none"></div>
```

Comment:
The `fullscreen` div is a blur to prevent the user from interacting with the app while the wrong eth network is selected, so it has the same behavior as the banner with `unsupportedNetworkBanner`

Reply:
The blur is on top of the banner?

Reply:
No it covers everything under the banner

Reply:
by under I mean below, i.e the app functionality